# config/zedm_yaml
# Parameters for Stereolabs ZED mini camera
---

/**: # See https://github.com/yossioo/ros2-yaml-unnamed-node
    ros__parameters:
        general:
            camera_model:       'zed2'
        depth:
            min_depth:          0.3               # Min: 0.3, Max: 3.0
            max_depth:          10.
        imu:
            imu_frame:          'zed_imu_link'
            imu_pub_rate:       100.0
            imu_sync_frame:     false
            qos_history:        0                 # '0': KEEP_LAST - '1': KEEP_ALL
            qos_depth:          10                # Queue size if using KEEP_LAST
            qos_reliability:    1                 # '0': BEST_EFFORT - '1': RELIABLE
            qos_durability:     1                 # '0': TRANSIENT_LOCAL - '1': VOLATILE
